packages/dtc/makefile.patch

```diff
@@ -0,0 +1,18 @@
+--- dtc-1.4.7/Makefile	2018-12-24 00:56:22.322225248 +0530
++++ dtc-1.4.7.mod/Makefile	2018-12-24 00:57:22.645564591 +0530
+@@ -25,13 +25,13 @@
+ SWIG = swig
+ PKG_CONFIG ?= pkg-config
+ 
+-INSTALL = /usr/bin/install
++INSTALL = @TERMUX_PREFIX@/bin/install
```

Comment:
Any reason to use $PREFIX/bin/install ? For me, it fails:
```
	 INSTALL-BIN
make: /data/data/com.termux/files/usr/bin/install: Command not found
make: *** [Makefile:201: install-bin] Error 127
```
\- never use tools from /data/data directory, they are either may not exist or may be for different CPU architecture.
```suggestion
+INSTALL = /usr/bin/install
```

Reply:
It was working for me

Reply:
> It was working for me

on device but not in docker image.